x-pack/plugins/security/server/elasticsearch/index.ts

```diff
@@ -4,6 +4,9 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { AuthenticatedUser } from '../../common/model';
+
+export type Authentication = Omit<AuthenticatedUser, 'authentication_provider'>;
```

Comment:
__note:__ not super happy about the name of this type, but don't have any other ideas what the type name of `authentication` response field could be ....

Reply:
For reference, we discussed the naming offline and decided to name this field `AuthenticationInfo`. 

While being short, `Authentication` is slightly meaningless and could mean many things to different people. 

According to the ES docs the authentication field represents:

> User information such as their username, the roles that are assigned to the user, any assigned metadata, and information about the realms that authenticated and authorized the user

So renaming this type to `AuthenticationInfo` makes that a bit clearer.